airbyte-integrations/connectors/source-mongodb-v2/src/main/java/io/airbyte/integrations/source/mongodb/state/MongoDbStateManager.java

```diff
@@ -290,28 +301,24 @@ public AirbyteStateMessage createFinalStateMessage(final ConfiguredAirbyteStream
     if (stream.getSyncMode() == INCREMENTAL) {
       final AirbyteStreamNameNamespacePair pair = new AirbyteStreamNameNamespacePair(stream.getStream().getName(), stream.getStream().getNamespace());
       if (!streamPairToLastIdMap.containsKey(pair)) {
+        // fix me!
```

Comment:
??

Reply:
sorry forgot to cleanup. was thinking about creating a map but ended up not using it